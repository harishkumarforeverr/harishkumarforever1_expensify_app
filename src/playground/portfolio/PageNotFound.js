import React from 'react';
import { Link } from 'react-router-dom';
const PageNotFound =()=>(
    <div> 
      <h1> 404 </h1> 
      <h2> <Link to ="/"> GO Home </Link> </h2>
    </div>
)
export default PageNotFound;