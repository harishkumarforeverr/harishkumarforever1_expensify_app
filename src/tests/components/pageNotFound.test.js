import React from "react"; 
import {shallow} from "enzyme";
import PageNotFound from "../../components/pageNotFound"; 

test("test/component/PageNotFound  :::: should render the componet nicely",()=>{
    const wrapper=shallow(<PageNotFound/>);
    expect(wrapper).toMatchSnapshot();
})