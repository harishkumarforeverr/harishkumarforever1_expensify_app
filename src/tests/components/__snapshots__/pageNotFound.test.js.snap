// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test/component/PageNotFound  :::: should render the componet nicely 1`] = `
<div>
  <h4>
    404!
  </h4>
  <Link
    to="/"
  >
    Go Home
  </Link>
</div>
`;
